.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "ocluster"]
+	path = ocluster
+	url = https://github.com/ocurrent/ocluster.git
```

Comment:
You should add the branch name for each of these submodules.
`	branch = master`